### PR TITLE
[ePD] Connect FR, Amendments, Signature&Dates, Partner Reporting Requirements to permissions.view

### PR DESCRIPTION
**Pull request by roman-karpovich**
[ch25946]

### CODE REVIEW

File: src/etools/applications/partners/permission_matrix/intervention_permissions.csv [hunk 1]
```diff
@@ -50,8 +52,9 @@ Field no,Field Name,Group,Condition,Status,Action,Allowed
 3.9.6,flat_locations,Unicef Focal Point,user_adds_amendment,Signed,Edit,TRUE
 3.9.7,flat_locations,Unicef Focal Point,contingency on,Signed,Edit,TRUE
 3.9.7,flat_locations,Unicef Focal Point,user_adds_amendment,Active,Edit,TRUE
-3.9.8,frs,UNICEF User,not_in_amendment_mode,*,View,TRUE
-3.9.8,frs,Unicef Focal Point,,Draft,Edit,TRUE
+3.9.8,frs,Partner User,,*,View,FALSE
+3.9.8,frs,*,user_adds_amendment,*,View,FALSE
+3.9.8,frs,*,,Draft,View,FALSE
```
Comment (robertavram) [inline src/etools/applications/partners/permission_matrix/intervention_permissions.csv:57]: frs should be viewable in draft

File: src/etools/applications/partners/tests/test_api_interventions.py [hunk 2]
```diff
@@ -116,7 +116,6 @@ class TestInterventionsAPI(BaseTenantTestCase):
             "equity_narrative",
             "equity_rating",
             "flat_locations",
-            "frs",
```
Comment (robertavram) [inline src/etools/applications/partners/tests/test_api_interventions.py:119]: frs should be editable in draft, review, signature and signed

File: src/etools/applications/partners/tests/test_epd_management/test_funds_reservation.py [hunk 3]
```diff
@@ -13,7 +13,7 @@ class TestFundsReservationManagement(BaseTestCase):
     def test_unicef_permissions(self):
         response = self.forced_auth_req(
             'get',
-            reverse('pmp_v3:intervention-detail', args=[self.draft_intervention.pk]),
```
Comment (robertavram) [inline src/etools/applications/partners/tests/test_epd_management/test_funds_reservation.py:16]: keep it a draft for FRs

File: src/etools/applications/partners/tests/test_epd_management/test_funds_reservation.py [hunk 4]
```diff
@@ -23,7 +23,7 @@ def test_unicef_permissions(self):
     def test_partnership_manager_permissions(self):
         response = self.forced_auth_req(
             'get',
-            reverse('pmp_v3:intervention-detail', args=[self.draft_intervention.pk]),
```
Comment (robertavram) [inline src/etools/applications/partners/tests/test_epd_management/test_funds_reservation.py:26]: keep at draft

File: src/etools/applications/partners/tests/test_epd_management/test_funds_reservation.py [hunk 5]
```diff
@@ -46,7 +46,7 @@ def test_sync(self, sync_mock):
         def generate_frs(business_area_code, fr_number):
             FundsReservationHeaderFactory(
                 fr_number=fr_number,
-                intervention=self.draft_intervention,
```
Comment (robertavram) [inline src/etools/applications/partners/tests/test_epd_management/test_funds_reservation.py:49]: keep at draft everywhere, FRs should be editable in draft

File: src/etools/applications/partners/permission_matrix/intervention_permissions.csv [hunk 6]
```diff
@@ -50,8 +52,9 @@ Field no,Field Name,Group,Condition,Status,Action,Allowed
 3.9.6,flat_locations,Unicef Focal Point,user_adds_amendment,Signed,Edit,TRUE
 3.9.7,flat_locations,Unicef Focal Point,contingency on,Signed,Edit,TRUE
 3.9.7,flat_locations,Unicef Focal Point,user_adds_amendment,Active,Edit,TRUE
-3.9.8,frs,UNICEF User,not_in_amendment_mode,*,View,TRUE
-3.9.8,frs,Unicef Focal Point,,Draft,Edit,TRUE
```
Comment (robertavram) [inline src/etools/applications/partners/permission_matrix/intervention_permissions.csv:54]: keep edit true for draft for FRs

**Review by robertavram — CHANGES_REQUESTED** [review]
frs should be viewable and editable by unicef in draft.

**Comment by roman-karpovich** [comment]
reverted frs-related changes

**Pull request merged**